scripts/monorepo/src/getAffectedPackages.js

```diff
@@ -10,14 +10,23 @@ const { workspaceRoot } = require('./utils');
  * @returns {Set<string>} - Set of packages that are affected by in the current branch
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33103&commitId=6bce915697afa7e967667f5d75b8528af25381d7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|6| Changed|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|974| Changed|
|Drawer.overlay drawer full.chromium.png|2903| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>